common/src/nfts.rs

```diff
@@ -14,10 +14,13 @@ pub trait NFTs {
     /// How NFTs are represented internally.
     type NFTId: Parameter + Copy;
 
+    type NFTSeriesDetails;
+
     /// Create a new NFT with the specified details and return its ID or an error.
     fn create(
         owner: &Self::AccountId,
         details: Self::NFTDetails,
+        series_details: Option<Self::NFTSeriesDetails>,
```

Comment:
Why an `Option`?

Reply:
Good question :)

I got the felling if someone wants to create just one nft that there is no need for him to handle the additional parameters that are only used if a series is supposed to be created. Of course, I might be working on wrong assumptions about how the interface is going to be consumed by the caller.

Reply:
It is probably a responsibility for the UI, not really for us. Maybe we can have a separate serie for NFTs with no series?

Reply:
That's a good idea too. Like reserving series_id 0 (or the MAX value) as the id for all the nfts that are special. My only concern is that additional reads and writes are going to happen then which is not something that we want.